tests/test_e_pmv.py

```diff
@@ -20,10 +20,7 @@ def test_e_pmv(get_e_pmv_url, retrieve_data, is_equal):
         )
 
         try:
-            if isinstance(expected_outputs, list):
-                np.testing.assert_equal(result, expected_outputs)
-            else:
-                assert is_equal(result, expected_outputs)
+            assert is_equal(result, expected_outputs, tolerance.get("pmv", 1e-6))
```

Comment:
please ensure consistency because sometimes you write the `key`name and sometimes you pass the key name as a string. I think the option with the key is better